.github/workflows/release-contracts.yml

```
@@ -0,0 +1,51 @@
+name: Release contracts
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+      - ci/release-contracts
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    container: cosmwasm/workspace-optimizer:0.12.11
```

```suggestion
    container: cosmwasm/workspace-optimizer:0.12.13
```

This is the version we have currently in the repo, will bump in the future.